Shared/Samples/Create load report/CreateLoadReportView.Model.swift

```diff
@@ -91,11 +69,11 @@ extension CreateLoadReportView {
         private func loadUtilityNetwork() async throws {
             statusText = "Loading utility network…"
             try await utilityNetwork.load()
-            statusText = nil
+            defer { statusText = nil }
             
             guard let startingLocation = makeStartingLocation(),
                   // Get the base condition and trace configuration from a default tier.
-                  let traceConfiguration = getTraceConfiguration() else { return }
+                  let traceConfiguration = getTraceConfiguration() else { throw LoadError.cannotLoadNetwork }
```

Comment:
The error is called `cannotLoadNetwork`, but the network did load or the call on line 71 would have thrown an error.

Reply:
alternatively you can have the dependent functions be throwing and return non optionals:

```swift
private func makeStartingLocation() throws -> UtilityElement
```

And they can throw something like:

```
enum Error {
    case startingLocationNotFound
    case traceConfigurationNotFound
}
```